packages/driver/src/cy/commands/agents.ts

```diff
@@ -68,7 +68,7 @@ const onInvoke = function (Cypress, obj, args) {
   const logProps: Record<string, any> = {
     name: agentName,
     message: obj.message,
-    error: obj.error,
+    state: obj.error ? 'failed' : 'passed',
```

Comment:
Allows the command queue to handle reporting the error. This is a weird one in general because of how stubs can be created. Fake call throughs with assertions are where this was needed. It's odd because this is logged after the sync func completes, but if it's an async func or a stubbed API this appears to be fine.
Given:
```js
 validate.callsFake(() => {
    expect(true).to.be.false
  })

cy.sessoin('id', () =>, { validate })
```

| Without Change | With Change |
| - | - |
| <img width="585" alt="Screen Shot 2022-11-03 at 8 49 12 AM" src="https://user-images.githubusercontent.com/14099737/199738738-d3e2f972-2aa0-40b2-a495-2d06ca784e0c.png">|<img width="588" alt="Screen Shot 2022-11-03 at 8 49 41 AM" src="https://user-images.githubusercontent.com/14099737/199738741-b0c79569-99a7-4d9b-9d22-92c4f4bb0ec5.png">